### DISCUSSION TITLE
请教：ColossalAI 和 微软开源的DeepSpeed chat 的定位差异、优势是什么，谢谢

### DISCUSSION THREAD

**Discussion by yanni-code** [hpcaitech/ColossalAI#3843] [Community | Q&A]
你好！请教个问题，ColossalAI 和 微软开源的DeepSpeed chat 的定位差异、优势是什么？ 因为我分别了看了ColossalAI 和 DeepSpeed chat的公开的比较材料都在说比对方的性能好，非常困惑，还请抽空解答一下，非常感谢。

**Comment by AntyRia** [comment]
据我用的结果来看 DeepSpeed chat在张量并行方面做的一般，ColossalAI主要在原来的数据并行、流水线并行基础上进一步强化了模型并行的张量并行部分

**Comment by ppt0011** [comment]
之前上手train resnet的时候觉得colossal AI更user friendly, 详见[medium](https://ppt001.medium.com/training-resnet-with-parallel-and-distributed-training-frameworks-pytorch-ddp-deepspeed-and-881e3433ab11)

**Comment by SeekPoint** [comment]
好像唯一的区别/优点就是基于chunk的异构内存管理，但是效果也不是特别明显

个人粗浅理解，望雅正
